intro2.md

```diff
@@ -0,0 +1,405 @@
+
+### 🎶 All together now 🎶
+
+#### Optimistic Ethereum is an _EVM equivalent_, _optimistic rollup_ protocol designed to _scale Ethereum_.
+
+## Network Participants
+
+There are three actors in Optimistic Ethereum: users, sequencers, and verifiers.
+
+<!--
+![Network Overview](./assets/network-participants-overview.svg)
+-->
+
+![Network Overview](https://raw.githubusercontent.com/ethereum-optimism/optimistic-specs/main/assets/network-participants-overview.svg)
+
+### Users
+
+At the heart of the network are users (us!). Users can:
+
+1. Deposit or withdraw tokens by sending transactions to Ethereum mainnet.
+2. Send transactions to the sequencer, to use EVM smart contracts on L2, or to
+   send ETH to other L2 users.
+3. View the status of transactions using block explorers provided by network
+   verifiers.
+
+### Sequencer
+
+As a first approximation, you can see the sequencer as the L2 block producer. It
+accrues user transactions into "batches" and submits these batches to contracts
+on L1 in order to make L2 blocks out of them (beware however, that there isn't a
+1-1 mapping between batches and blocks). Unlike on L1, on L2 transactions can be
+confirmed before a full block (or even a full batch) is created!
+
+We are going to give a lot more detail about the operation of the sequencer in
+the rest of this document.
+
+### Verifiers
+
+Verifiers monitor L1 for rollup data. They serve three purpose:
+
+1. Serving rollup data to users; and
+2. Verifying rollup integrity and disputing invalid assertions.
+3. Propagate the L2 state among validators (\*)
+
+In order for the network to remain secure there must be at least one honest
+verifier who is able to verify the integrity of the rollup chain & serve
+blockchain data to users.
+
+(\*) It's important to note that a validator does not rely on this data to
+validate the rollup — only access to the L1 chain is required. However, getting
+the most recent L2 state allows validators to serve the state to users, and to
+pre-validate transactions that should soon be posted to L1. L2 state propagation
+uses the same mechanism as L1 state sync, but must additionally include a
+signature from the sequencer to ensure the legitimacy of the data.
+
+## User Transactions and The Sequencer
+
+As a user, once you have [bridged] (\*) some ETH over to Optimistic Ethereum,
+you will probably interact with it though your wallet, such as Metamask. Add the
+network (chainID 10, [JSON-RPC] endpoint `https://mainnet.optimism.io/`) and
+you're good to go.
+
+(\*) L1 -> L2 deposits will be explained later.
+
+[bridged]: https://www.optimism.io/apps/bridges
+[JSON-RPC]: https://github.com/ethereum/execution-apis
+
+When you send a transaction, it is sent to the node called the sequencer. The
+sequencer will run verify your transaction, execute it if valid, and confirm it.
+This happens much faster than on Ethereum mainnet (around ~1s).
+
+There is currently a single sequencer operated by Optimism PBC. We expect future
```

Comment:
If we are going to mention that there is a single sequencer operated by the company, then we should mention that the profits are being used to fund public goods to prevent the conflict of interest where there is no incentive to actually decentralize. While this is related to the threat model, its unclear how related it is to the protocol specification

Reply:
The specification will assume a single sequencer, so that should be mentionned. Should I just remove "operated by Optimism PBC"? That is indeed not related to the spec (as someone else can implement it.)